src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -32,39 +30,45 @@ namespace Uno.Toolkit.UI
 	public static class InputExtensions
 	{
 		private static readonly ILogger _logger = typeof(InputExtensions).Log();
-
-		public enum ReturnType {
-			Default,
-			Done,
-			Go,
-			Next,
-			Search,
-			Send
-		}
-
 		#region DependencyProperty: ReturnType
 
+#if HAS_UNO_WINUI
```

Comment:
why is that wrapped within HAS_UNO_WINUI?
for stuff accessible within xaml, it is preferable that you keep them in all cases
and just do-nothing on the not-supported platforms